src/transformers/models/falcon_mamba/modeling_falcon_mamba.py

```diff
@@ -45,8 +45,10 @@
     pscan = None
 
 if is_mamba_ssm_available():
-    from mamba_ssm.ops.selective_scan_interface import mamba_inner_fn, selective_scan_fn
+    from mamba_ssm.ops.selective_scan_interface import selective_scan_fn
     from mamba_ssm.ops.triton.selective_state_update import selective_state_update
+
+    from ...kernels.falcon_mamba import mamba_inner_fn
 else:
     selective_state_update, selective_scan_fn, mamba_inner_fn = None, None, None
```

Comment:
I think we should check for causal conv1d here as well. It's been optional at some point and the `mamba_inner_fn` depends on it to be available.

Reply:
Nvm, I forgot that there is an `all` check later in the file so it's not necessary.